docs/backtesting.md

```diff
@@ -3,7 +3,7 @@
 This page explains how to validate your strategy performance by using
 Backtesting.
 
-## Getting data for backtesting / hyperopt
+## Getting data for backtesting and hyperopt
 
 To download backtesting data (candles / OHLCV), we recommend using the `freqtrade download-data` command.
```

Comment:
"You can download historical data (candles / OHLCV) for backtesting and hyperoptimization by means of the `freqtrade download-data` command."